style.css

```diff
@@ -425,3 +432,234 @@ footer {
+    width: 100%;
+    border: none;
+  }
+
+  #works h1::after {
+    content: '';
+    position: absolute;
+    top: 40%;
+    background-color: #212121;
+    height: 1px;
+    width: 70%;
+    right: 0;
+  }
+
+  .projects {
+    margin-top: 90px;
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  .multi_stories {
+    grid-column-start: 1;
+    grid-column-end: 4;
+    flex-direction: row;
+    height: 448px;
+  }
+
+  .multi_stories_content {
+    margin-left: 24px;
+  }
+
+  .multi_stories_content h3 {
+    font-size: 40px;
+    line-height: 52px;
+    margin-top: 0;
+    margin-bottom: 24px;
+  }
+
+  .multi_stories_content p {
+    margin-bottom: 24px;
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .multi_stories_content ul {
+    margin-top: 0;
+    margin-bottom: 24px;
+  }
+
+  .multi_stories_content button {
+    margin-top: 0;
+  }
+
+  .project {
+    height: 435px;
+    width: 369px;
+    cursor: pointer;
+    overflow: hidden;
+    background-image: url('https://s3-alpha-sig.figma.com/img/7e29/831a/679f12c86bb033399c6b604b71698f3c?Expires=1675641600&Signature=HQZ90jxbjfrJ4rEp485YY1X9P0mXO6hL2M-H7mA~OKkYV07hWsP0qNbs1RGdYyoebNP9SpJws8OvqIV-pW7RGvkjENYQS1Yp58j7wIQ3cZg3F0IiAGuCALtPurK1EXjSp4JRN4-S0g4HabuAMRXhJF-WHNALhNsq5UgSvfdxbVqBfMM6y5Hxb38NUITsUFb4~Z7NLrTMB4ru~y2nAsbedf4dEV2NZZIj6TGFPY8mTL5mTnh5~28ERadEsPpPY3lHcnzqP4TcMH988ubbCWZDWR10nfBes-rUndASKTIPi1vKZZ74J42RpkkvGQakv1RftflSNJ3F00w7PIC07CVtgw__&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4');
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+  }
+
+  .project button {
+    display: none;
+    transition: all 0.3s ease-in-out;
+    transform: translateY(100%);
+  }
+
+  .project_info {
+    transition: all 0.5s ease-in-out;
+    padding: 16px;
+  }
+
+  .project:hover .project_info {
+    transform: translateY(-100%);
+  }
+
+  .project:hover button {
+    display: block;
+    transform: translateY(0);
+  }
+
+  .project_info ul {
+    margin-top: 24px;
+  }
+
+  .project_info p {
+    margin-top: 24px;
+    font-size: 17px;
+  }
+
+  #about {
+    padding: 130px 142px;
+    background-image: url('https://res.cloudinary.com/david79/image/upload/v1674579763/about_me_bg_jypfej.png');
+    background-repeat: no-repeat;
+    background-position-x: right;
+    background-position-y: 99px;
+  }
+
+  .about_me_content {
+    width: 566px;
+  }
+
+  .about_me_content p {
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .lang_skill_frame {
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: space-between;
+  }
+
+  .category_heading h3 {
+    line-height: 44px;
+  }
+
+  .cate_list {
+    width: 260px;
+    align-items: flex-start;
+  }
+
+  .cate_list ul {
+    align-items: flex-start;
+    margin: 36px 0;
+  }
+
+  #contact {
+    padding: 130px 135px;
+    flex-direction: row;
+  }
+
+  #contact h4 {
+    width: 455px;
+    font-size: 40px;
+    line-height: 52px;
+    margin-right: 24px;
+  }
```

Comment:
- [ ] I like the way that you set your elements inside the last section :+1:, but I have noticed that the title of the contact me section should be aligned to the left to match with the Figma template :+1: 

**Your design**

![Screenshot from 2023-01-26 07-58-21](https://user-images.githubusercontent.com/105117832/214846412-76cf44e6-79fd-409f-9303-2a1d615db144.png)

**Figma template**

![Screenshot from 2023-01-26 07-58-35](https://user-images.githubusercontent.com/105117832/214846480-f38fa848-264c-453f-a693-a7e95319a75b.png)